src/Common/hooks/useNotificationSubscribe.ts

```diff
@@ -0,0 +1,131 @@
+import { useDispatch } from "react-redux";
+import {
+  getUserPnconfig,
+  updateUserPnconfig,
+  getPublicKey,
+} from "../../Redux/actions";
+import { useState } from "react";
+import * as Sentry from "@sentry/browser";
+import { useTranslation } from "react-i18next";
+import { Error } from "../../Utils/Notifications.js";
+import useAuthUser from "../../Common/hooks/useAuthUser";
+
+export default function useNotificationSubscribe() {
+  const { username } = useAuthUser();
+  const dispatch: any = useDispatch();
+  const [isSubscribed, setIsSubscribed] = useState("");
```

Comment:
The naming seems to be bit off, why not change it to some thing like `subscriptionStatus`. 

Also add `type subscriptionStatus = "NotSubscribed" | "SubscribedOnAnotherDevice" | ...` instead of just having generic string type for state

Reply:
done